packages/manager/apps/dedicated/client/app/ip/components/list/popover/ipBlock.html

```diff
@@ -112,5 +112,26 @@
                 data-ng-bind=":: ('ip_table_manage_delete_ipblock' | translate) + ADDITIONAL_IP"
             ></button>
         </li>
+
+        <!-- BYOIP Slice -->
+        <li data-ng-if="ipBlock.bringYourOwnIp && !ipBlock.ipBlock.endsWith('/32')">
+            <button
+                type="button"
+                class="btn btn-link"
+                data-ng-click="setAction('ip/byoip/slice/ip-ip-byoip-slice', { ipBlock: ipBlock })"
```

Comment:
Instead of using setAction, as I suggested you you should create a new state, this way you could follow the standards and simplify code for the ip slice controller

Reply:
As slicing and aggregation are done in components (used in both ip lists) it is too complex to set up routing for these (would require two states to keep the display on top of each lists).